Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1952,10 +1952,12 @@ private enum ExceptionMode {
                     PythonThread bpThread = null;
                     bool wrongLine = false;
                     proc.BreakpointHit += (sender, args) => {
+                        var continueEvent = new AutoResetEvent(true);

```

Comment:
Surely it can be scoped to just around the four lines of code that run between the Reset() and the wait? We only ever set it in one branch, and it's an auto-reset, so presumably we're waiting 1s in most cases when we don't need to be.

Now's as good a time as any to clean up all the tests.